sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/pipeline/_io/base.py

```diff
@@ -430,7 +430,6 @@ def version(self, version):
         """Receive input version,
         assign the version to NodeOutput/PipelineOutput and build data according to the version"""
         self._build_default_data()
-        self._data.type = self.type
```

Comment:
Any idea why we update type in a version setter before?

Reply:
This line is added by me, I think I add it wrongly😅 and we should delete this line.